README.md

```diff
@@ -34,6 +34,7 @@ Note: grab-site currently **does not work with Python 3.5**; please use Python 3
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
```

Comment:
This comment is a lie, sorry. I've been updating this TOC manually and probably don't want the `Tips for specific websites` expanded.